Exiled.API/Features/Player.cs

```diff
@@ -2038,14 +2038,14 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
         public GameObject PlaceTantrum() => Map.PlaceTantrum(Position);
 
         /// <summary>
-        /// Give an new AhpProccess.
-        /// </summary>
-        /// <param name="amount">The ammount of AHP he get.</param>
-        /// <param name="limit">The maximum AHP he can get.</param>
-        /// <param name="decay">How many it's lost per tick.</param>
-        /// <param name="efficacy">percentage of damage it's take on Hurt.</param>
-        /// <param name="sustain">Time before the AHP is decay.</param>
-        /// <param name="persistant">The Proccess it's not deleted when ahp it's at 0.</param>
+        /// Gives a new.
```

Comment:
Gives the player a new AHP Process.

Reply:
if i put 
/// Gives a new <see cref="AhpStat">adding ahp</see>.
is it good ?

Reply:
No, that's also not proper english.
Just put what I said.

Reply:
Okay